downloadaudio/download.py

```diff
@@ -50,6 +51,9 @@
 from .review_gui import review_entries
 from .update_gui import update_data
 
+DOWNLOAD_NOTE_SHORTCUT  = "q"
```

Comment:
Are you sure that this will not be triggered any time you press q, regardless of context? Can you still type the letter q into a type-the-answer input box?